sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -118,9 +118,10 @@ def test_operator_to_template(self):
     self.maxDiff = None
     self.assertEqual(golden_output, compiler.Compiler()._op_to_template(op))
 
-  def _get_yaml_from_tar(self, tar_file):
-    with tarfile.open(tar_file, 'r:gz') as tar:
-      return yaml.load(tar.extractfile(tar.getmembers()[0]))
+  def _get_yaml_from_zip(self, zip_file):
```

Comment:
Can you add basic tests for covering yaml and tar.gz formats here?

Reply:
Done.